lib/index.js

```diff
@@ -52,10 +52,12 @@ Mouseflow.prototype.loaded = function() {
  * @param {Page} page
  */
 
-Mouseflow.prototype.page = function() {
-  if (!window.mouseflow) return;
-  if (typeof window.mouseflow.newPageView !== 'function') return;
-  window.mouseflow.newPageView();
+Mouseflow.prototype.page = function(page) {
+  if (page.properties() && page.properties().path) {
+    push('newPageView', page.properties().path);
```

Comment:
to remove `if` and use `page.path()` which basically looks up same thing: https://github.com/segmentio/facade/blob/master/lib/page.js